components/tyk-enterprise-portal/templates/statefulset-enterprise-portal.yaml

```diff
@@ -0,0 +1,235 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "tyk-enterprise-portal.fullname" . }}
+  labels:
+    app: {{ include "tyk-enterprise-portal.fullname" . }}
+    chart: {{ include "tyk-enterprise-portal.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  serviceName: service-{{ include "tyk-enterprise-portal.fullname" . }}
```

Comment:
Name doesn't match with the service name defined in `service-enterprise-portal.yaml`

Reply:
Agree. Due to this, service has no control over pods, so we cannot access portal through service name (or port-forward)

Reply:
Fixed